src/main/java/org/opensearch/knn/index/query/KNNQueryBuilder.java

```diff
@@ -72,6 +76,17 @@ public KNNQueryBuilder(String fieldName, float[] vector, int k) {
         this(fieldName, vector, k, null);
     }
 
+    /**
+     * Constructs a new query for distance type radius search
+     *
+     * @param fieldName Name of the filed
+     * @param vector Array of floating points
+     * @param distance Distance threshold for the given vector
```

Comment:
Can you add more detail here for distance parameter threshold. For instance, "if passed, any result whose distance based on the space type being used will not be returned."